config/webpack.base.config.js

```diff
@@ -23,25 +23,32 @@ const baseConfig = {
     // provide much cleaner feedback while building on the command line
     new WebpackBar(),
     new webpack.DefinePlugin({
-      MC_VERSION: JSON.stringify(require(path.resolve(basedir, 'package.json')).version),
+      MC_VERSION: JSON.stringify(
+        require(path.resolve(basedir, 'package.json')).version
+      ),
       'process.env': {
-        'SENTRY_DSN': JSON.stringify(process.env.SENTRY_DSN || ''),
-        'SENTRY_ENVIRONMENT': JSON.stringify(process.env.NODE_ENV || 'development'),
-        'SUPPORT_URL': JSON.stringify(process.env.SUPPORT_URL || ''),
-        'INQUIRY_EMAIL': JSON.stringify(process.env.INQUIRY_EMAIL || ''),
-     },
+        SENTRY_DSN: JSON.stringify(process.env.SENTRY_DSN || ''),
+        SENTRY_ENVIRONMENT: JSON.stringify(
```

Comment:
isn't `JSON.stringify` redundant here

Reply:
mmh actually this was form the existing codebase, (only prettier formatted the file after I added my config) I think it boils down to how the `.env `variables are defined. Since the [define-plugin](https://webpack.js.org/plugins/define-plugin/) does direct text replacement, I was under the impression all of these env variables would not include any types of quotes